lib/localize.js

```diff
@@ -129,12 +158,25 @@ export const getLocalizeClass = (superclass = class {}) => class LocalizeClass e
 			const possibleLanguages = this._generatePossibleLanguages(config);
 			const resourcesPromise = this.getLocalizeResources(possibleLanguages, config);
 			resourcesLoadedPromises.push(resourcesPromise);
+			if (config?.loadCommon) {
+				resourcesLoadedPromises.push(this.getLocalizeResources(possibleLanguages, {
+					importFunc: async(lang) => {
+						if (commonResources.has(lang)) return commonResources.get(lang);
+						const resources = (await import(`../lang/${lang}.js`)).default;
+						commonResourcesImportCount++;
+						commonResources.set(lang, resources);
+						return resources;
+					}
+				}));
+			}
 		}
 		return Promise.all(resourcesLoadedPromises);
 	}
 
 	static async _getLocalizeResources(langs, { importFunc, osloCollection, useBrowserLangs }) {
 
+		if (importFunc === undefined) return;
```

Comment:
Change 2: the default implementation of `getLocalizeResources` just calls this, but `importFunc` can now be `undefined`. So we short-circuit and return `undefined`, but that's going to cause change 3 below...